src/modules/esl-event-listener/core/targets/inert.target.event.ts

```diff
@@ -0,0 +1,43 @@
+import {overrideEvent} from '../../../esl-utils/dom/events/misc';
+
+/**
+ * Event names that could be triggered by {@link ESLWheelTarget}
+ */
+export type WheelEventName = 'longwheel';
+
+/**
+ * Describes scroll information provided with {@link ESLWheelTarget}
+ */
+export interface ESLWheelEventInfo {
+  /** Distance scrolled along the x-axis */
+  deltaX: number;
+  /** Distance scrolled along the y-axis */
+  deltaY: number;
+  /** First wheel event */
+  startEvent: WheelEvent;
+  /** Last wheel event */
+  endEvent: WheelEvent;
+}
+
+/**
+ * Wheel event dispatched by {@link ESLWheelTarget}
+ */
+export class ESLWheelEvent extends UIEvent implements ESLWheelEventInfo {
+  public override readonly target: Element;
+
+  public readonly deltaX: number;
+  public readonly deltaY: number;
+  public readonly endEvent: WheelEvent;
+  public readonly startEvent: WheelEvent;
+
+  protected constructor(eventName: WheelEventName, target: Element, wheelInfo: ESLWheelEventInfo) {
+    super(eventName, {bubbles: true, cancelable: true});
```

Comment:
1. For a customly-detected sequence of events, the meaning of 'cancel' seems unclear. As no valid definition exists, the event should not be considered cancelable.

2. This custom event is dispatched on a SyntheticEventTarget instance, where there is no definition or capability for such an event to propagate or bubble.